CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ Please keep one empty line before and after all headers. (This is required for `
 And please only add new entries to the top of this list, right below the `# Unreleased` header.
 
 # Unreleased
+- **Breaking** Renamed `EventLoopWindowTarget::set_device_event_filter` to `EventLoopWindowTarget::listen_device_events` and `DeviceEventFilter` to `DeviceEvents` and appropriately reversed its variants.
+- On X11, fix `EventLoopWindowTarget::listen_device_events` effect being reversed.
 
```

Comment:
```suggestion
# Unreleased

- **Breaking** Rename `DeviceEventFilter` to `DeviceEvents` reversing the behavior of variants.
- **Breaking** Rename `EventLoopWindowTarget::set_device_event_filter` to `listen_device_events`.
- On X11, fix `EventLoopWindowTarget::listen_device_events` effect being reversed.
```

Reply:
Ensure that there's no empty line with the `**Breaking** below and there's one empty line to section heading.

Reply:
The line is too long, I've suggested the splitting in the `suggested change` but it's ignored.